src/cis/fam_cis_client.cpp

```diff
@@ -622,7 +623,7 @@ int Fam_CIS_Client::get_atomic(uint64_t regionId, uint64_t srcOffset,
 
 int Fam_CIS_Client::put_atomic(uint64_t regionId, uint64_t srcOffset,
                                uint64_t dstOffset, uint64_t nbytes,
-                               uint64_t key, const char *nodeAddr,
+                               uint64_t key, uint64_t srcBaseAddr, const char *nodeAddr,
```

Comment:
I do not see srcBaseAddr being used in this function, is it needed ? 
Here srdbaseaddress set from data : req.set_srcbaseaddr((uint64_t)data);

Reply:
In this case data & srcBaseAddr is same value. To make API parameter unified, we need to use srcBaseAddr. Changed now.